packages/fela-bindings/src/__tests__/ThemeProviderFactory-test.js

```diff
@@ -73,7 +73,8 @@ describe('Using the ThemeProvider', () => {
       }
 
       render() {
-        return this.props.children
+        const { children } = this.props
```

Comment:
Fixed (eslint rule prefer-template)

Reply:
This is due to the rule [react/destructuring-assignment](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/destructuring-assignment.md). NoUpdate is a synthetic component. In practice, it is very rare for a component to simply return its child. I think this is a useful rule and its use will lead to monotony in the code. Do not want to completely disable the rule because of one component. I can suppress this lint, but it will lead to a very long comment. Are you sure this is an unwanted check?

Reply:
Well I don’t see value in this plugin at all. It doesn’t add any benefits except for maybe higher readability in some cases, but here it decreases that and on top of that looks ugly.
If you want to keep the plugin, please disable it here and for all single property deconstructs.

Reply:
This rule does not support such detailed settings. I'll just turn it off.